README.md

```diff
@@ -553,6 +553,10 @@ export default Base.extend({
 });
 ```
 
+## Setting Current User as a part of the session
+
+[README for setting current user](https://github.com/simplabs/ember-simple-auth/blob/master/readmes/set-current-user.md)
+
```

Comment:
can we make this a relative link so it would link to the respective file in a branch when looking at that branch?